sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/CodeGenInternal/PatchModels/WholeMetricConfigurationPatch.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.AI.MetricsAdvisor.Models
+{
+    internal partial class WholeMetricConfigurationPatch
+    {
+        // Full qualification must be used so CodeGen skips the generation of this method.
+        // See: https://github.com/Azure/autorest.csharp/issues/793
+        void global::Azure.Core.IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
+        {
+            writer.WriteStartObject();
+            if (Optional.IsDefined(ConditionOperator))
+            {
+                writer.WritePropertyName("conditionOperator");
+                writer.WriteStringValue(ConditionOperator.Value.ToString());
+            }
```

Comment:
Not sending null on this one because the service does not support this scenario. They only accept "send value" / "not send" for this property.